tools/create-hybrid-image

```diff
@@ -24,41 +24,45 @@ DESC=
 HOMEPAGE=
 OUTPUT_DIR=.
 PROTOTYPE=
+ZFS_DIR=/data
 
 BUILD_DATE=$(date +%Y%m%d)
 
-SDC_VMMANIFEST=$(cd "$(dirname "$0")"; pwd)/sdc-vmmanifest
+ENGDIR=$(dirname "$(cd "$(dirname "$0")"; pwd)")
```

Comment:
huh - bash correctly nests quotes here because of the nested `$( ... )` ? TIL...

Reply:
Yep.  ksh does the same.